ynput/ayon-launcher
# Create server package command

## Changelog Description
Added command to prepare installer package ready for AYON server.

## Additional info
Did not test windows.

## Testing notes:
1. Make a build and installer
2. Run `./manage.ps1 create-server-package` or `./make.sh create-server-package`
3. Should create a zip file with installer and metadata file
